fail2ban/protocol.py

```diff
@@ -55,6 +55,7 @@
 ["start <JAIL>", "starts the jail <JAIL>"], 
 ["stop <JAIL>", "stops the jail <JAIL>. The jail is removed"], 
 ["status <JAIL>", "gets the current status of <JAIL>"],
+["status <JAIL> extended", "gets the current status of <JAIL> with extended info"],
```

Comment:
That was what I was originally thinking, but I like the idea of generalizing/abstracting it more.

Here is a sample output with some randomly generated IPs. RFC 1918 and multicast, etc come back with NXDOMAIN.

https://gist.github.com/anonymous/e09a9328564b30b555a6